src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class EmailValidatorTest {
+    private static final String EMAIL = "user@gmail.com";
```

Comment:
Consider using a more descriptive name for the constant EMAIL, such as VALID_EMAIL, to better express its purpose in the tests. 📝